ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3737,164 +3731,302 @@ AHU_CHWSC_ECON_FDPM_EDRPC_RFSS_RFVSC_SFSS_SFVSC_SSPC:
   implements:
-  - AHU_DX2SC_ECON_HT2SC_SFSS_SFVSC_SSPC
-  - INCOMPLETE
-  uses:
-  - zone_air_temperature_sensor
-  - zone_air_cooling_temperature_setpoint
-  - zone_air_heating_temperature_setpoint
+  - AHU
+  - SFSS
+  - SFMSC
+  - SFVSC
+  - FDPM
+  - CHWSC
+  - RTM
+  - RACO2C
+  - SSPC
+  - VOADM2X
+  - SAIDC2X
+  - RAIDC2X
 
-AHU_US_MTV_1210_5: # Appears changed in the controller.
-  id: "10056703581857251328"
-  description: "Non-standard type for 1210"
+AHU_SFSS_SFMSC_SFVSC_FDPM_CHWSC_RTM_RACO2C_SSPC2X_VOADM2X_SAIDC2X_RAIDC3X_SSPC:
+  id: "14811782409221046272"
+  description: "AHUs with chilled water temperature monitoring and compressor monitoring with zone temperature and humidity monitoring ."
+  is_canonical: true
   implements:
-  - REMAP_REQUIRED
-  uses:
-  - compressor_run_command
-  - economizer_mode
-  - heater_run_command
-  - mixed_air_temperature_sensor
-  - outside_air_damper_percentage_command
-  - return_air_temperature_sensor
-  - supply_air_static_pressure_setpoint
-  - supply_air_temperature_sensor
-  - supply_air_temperature_setpoint
-  - supply_fan_run_command
-  - supply_fan_run_status
-  - supply_air_flowrate_sensor
-  - zone_air_cooling_temperature_setpoint
-  - zone_air_heating_temperature_setpoint
+  - AHU
+  - SFSS
+  - SFMSC
+  - SFVSC
+  - FDPM
+  - CHWSC
+  - RTM
+  - RACO2C
+  - SSPC2X
+  - VOADM2X
+  - SAIDC2X
+  - RAIDC3X
+  - SSPC
 
-AHU_US_MTV_1210_7:
-  id: "12736345360142696448"
-  description: "Non-standard type for 1210"
+AHU_SFSS_SFMSC_SFVSC_FDPM_CHWSC_RTM_RACO2C_SSPC2X_VOADM2X_SAIDC2X_RAIDC2X_SSPC:
+  id: "18103913736828878848"
+  description: "AHUs with chilled water temperature monitoring and compressor monitoring with zone temperature and humidity monitoring ."
+  is_canonical: true
   implements:
-  - AHU_DX2SC_ECON_EFSS_HT2SC_SFSS_SFVSC_SSPC
-  - INCOMPLETE
-  uses:
-  - zone_air_cooling_temperature_setpoint
-  - zone_air_heating_temperature_setpoint
-  - zone_air_temperature_sensor
+  - AHU
+  - SFSS
+  - SFMSC
+  - SFVSC
+  - FDPM
+  - CHWSC
+  - RTM
+  - RACO2C
+  - SSPC2X
+  - VOADM2X
+  - SAIDC2X
+  - RAIDC2X
+  - SSPC
+
+
+###################################
+### Existing Non-standard Types ###
+###################################
 
-AHU_US_MTV_1210_9:
-  id: "12957021741883850752"
-  description: "Non-standard type for 1210"
+# TODO: Every AHU must have outside_air_temperature_sensor, so almost every AHU is currently incomplete.
+#       See b/148117417 for details
+
+AHU_UNDEFINED:
+  id: "8989855146007592960"
+  description: "An AHU with no telemetry data. Note this is non-canonical, as it is typically expected that telemetry will normally be available."
   implements:
-  - AHU_BYPSSPC_DX2SC_ECON_HT2SC_SFSS
-  - INCOMPLETE
-  uses:
-  - zone_air_heating_temperature_setpoint
-  - zone_air_cooling_temperature_setpoint
+  - AHU
 
-AHU_US_MTV_1220_4:
-  id: "2699440271849947136"
-  description: "Non-standard type for 1220"
+AHU_STDSPC_SSPC_EFFC_EFVSC_FDPM3X_HTWHLSTC_RHM_ETM:
```

Comment:
so here you are reusing a different type and therefore different id?

Reply:
dont compare line by line, because I had to move some types around. this one is just in a different spot, but ID is the same.

see previous version at line 4642